### PR TITLE
Add specs for kernel-monitor component

### PR DESCRIPTION
**Pull request by BenRussert**
- Test "jump to file" for both saved and unsaved editor links

### CODE REVIEW

**Comment by BenRussert** [comment]
We could add to this as needed. Should help with #1617.

**Comment by BenRussert** [comment]
xref #1601

**Comment by aviatesk** [comment]
The changes:
- Added a feature of showing kernel spec by clicking kernel display name
- Delete unused class name
- Added spec for restart/shutdown buttons

I still want to add tests for various other features, like kernel execution count / last exec time, but not sure to do for now.

File: spec/components/kernel-monitor-spec.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+import Adapter from "enzyme-adapter-react-16";
+const fs = require("fs");
+
+Enzyme.configure({ adapter: new Adapter() });
+
+import { Store } from "../../lib/store";
+import KernelTransport from "../../lib/kernel-transport";
+import Kernel from "../../lib/kernel";
+import KernelMonitor from "../../lib/components/kernel-monitor";
+import { waitAsync } from "../helpers/test-utils";
```
Comment (BenRussert) [inline spec/components/kernel-monitor-spec.js]: Side note: https://github.com/atom/atom/pull/18917

**Pull request merged**